sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -375,6 +376,66 @@ def test_radius_neighbors_classifier_outlier_labeling():
             clf.fit(X, y)
             assert_array_equal(correct_labels1, clf.predict(z1))
             assert_array_equal(correct_labels2, clf.predict(z2))
+            assert_array_equal(outlier_proba, clf.predict_proba(z2)[0])
+
+    # test outlier_labeling of using predict_proba()
+    RNC = neighbors.RadiusNeighborsClassifier
+    X = np.array([[0], [1], [2], [3], [4], [5], [6], [7], [8], [9]])
+    y = np.array([0, 2, 2, 1, 1, 1, 3, 3, 3, 3])
+
+    # test invalid outlier_label dtype
+    def check_exception():
+        clf = RNC(radius=1, outlier_label='a')
+        clf.fit(X, y)
+    assert_raises(TypeError, check_exception)
+
+    # test most frequent
```

Comment:
It wold be better design, I'll admit, to test the relationship between `predict_proba` and `predict` as a separate invariance test, while this function should focus on testing just one side of that invariance (i.e. what happens to one method's results when outlier_label is changed). But this is acceptable.